litellm/llms/bedrock_httpx.py

```diff
@@ -426,6 +429,18 @@ def get_credentials(
             client = boto3.Session(profile_name=aws_profile_name)
 
             return client.get_credentials()
+        elif (
+            aws_access_key_id is not None
+            and aws_secret_access_key is not None
+            and aws_session_token is not None
+        ): ### CHECK FOR AWS SESSION TOKEN ###
```

Comment:
do you need access_key + secret_access_key for session_token usage?

Reply:
yes, all 3.

Reply:
For reference, see: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html

Notably:
`Access key IDs beginning with ASIA are temporary credentials that are created using AWS STS operations`
and 
`Access key IDs beginning with AKIA are long-term credentials for an IAM user or an AWS account root user`

So when using session token, you get an `ASIA` prefixed secret key. And if you are using permanent fixed credentials, you secret key starts with `AKIA`